saleor/checkout/models.py

```diff
@@ -90,6 +90,9 @@ def __iter__(self):
     def __len__(self):
         return self.lines.count()
 
+    def get_customer_email(self):
```

Comment:
Do we really need this? Checkout is creating in API and forms. We should just confirm that `email` is always fulfilled. We could explicitly save it `checkout.save()`.